tests/Utils/Policies/UserPolicy.php

```diff
@@ -32,4 +32,9 @@ public function dependingOnArg($viewer, bool $pass): bool
     {
         return $pass;
     }
+
+    public function severalArgs($user, array $input): bool
+    {
+        return $input['key1'] === 'foo' && $input['key2'] === 'foo2' && $input['key2'] === 'foo3';
```

Comment:
you have a typo here. The last equal test should be with key3, not key2

Reply:
Hmm, then I probably needs to change the assert too